app/views/admin/articles/_form.html.erb

```diff
@@ -126,7 +126,8 @@
       <div class="col-xs-12 col-sm-6">
         <div class="form-group">
           <%= f.label :tweet %>
-          <%= f.text_area :tweet, class: "form-control" %>
+          <%= f.text_area :tweet, class: "form-control", data: { 'max-length': '115', 'feedback-box': 'tweet_length_feedback' } %>
+          <div id="tweet_length_feedback"></div>
```

Comment:
We use hyphens instead of underscores as word separators in attr values.
Please change these to that.